CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+
+We will then endeavour to respond to you as soon as possible.
+
+## I Want To Contribute
+
+> ### Legal Notice <!-- omit in toc -->
+> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the same licence as the existing content. We encourage the use of LLMs and other tools for improving readability of your drafts, however if a significant proportion of the content you are submitting is produced by generative AI this should be noted in your pull request.
+
+### Suggesting Improvements
+
+This section guides you through submitting a suggestion for improvements to the Reasonable Performance Computing SIG's Website & Knowledge-base, **including completely new guides or website features and minor improvements to existing content**. Following these guidelines will help maintainers and the community to understand your suggestion and find related suggestions.
+
+<!-- omit in toc -->
+#### Before Submitting an Idea
+
+- Check the [website](https://sig-rpc.github.io/), carefully to ensure your idea isn't redundant.
+- Perform a [search of existing issues](https://github.com/sig-rpc/sig-rpc.github.io/issues) to see if a similar suggestion has already been made. If it has, add a comment to the existing issue to show your support and add additional information, instead of opening a new one.
+- If you're suggesting a big change (more significant than a new guide), please ensure your idea fits with the scope and aims of SIG-RPC. It's up to you to make a strong case to convince the SIG's officers of the merits of your proposal.
+
+<!-- omit in toc -->
+#### How Do I Make a Good Suggestion?
+
+Suggested improvements are tracked as [GitHub issues](https://github.com/sig-rpc/sig-rpc.github.io/issues).
+
+- Select and follow the **[most appropriate issue template](https://github.com/sig-rpc/sig-rpc.github.io/issues/new/choose)**.
+- Use a **clear and descriptive title** for the issue to identify your suggestion.
+- Be **specific about the location** on the website that your improvement corresponds to.
+- **Explain why this improvement would be useful** to a significant fraction of users.
+- You may also want to point out **relevant websites/projects** that cover the suggested idea, which could serve as inspiration. 
+- If you think a dynamic feature of the website is broken or could be improved:
+  - **Describe the current behaviour** and **explain which behaviour you expected to see instead** and why.
+  - **Include the web browser you are using**, and any other hardware/software details that may be relevant (e.g. is it caused by an ad-blocker?).
+
+
+
+
+### Your First Guide Contribution
+
+Guides live under the `optimisations/` and `profilers/` directories respectively, grouped by language. Each guide is written in Markdown (`.md`) and should:
```

Comment:
Is it worth specifying that it's Jekyll/Kramdown markdown? (given there's a lot of variants, and the jekyll website is the target medium rather than viewing in github). 

I.e. `{:target="_blank"}` is a kramdown feature that won't behave consistently across other markdodn implemetnations?

Reply:
```suggestion
Guides live under the `optimisations/` and `profilers/` directories respectively, grouped by language. Each guide is written in the [Kramdown](https://kramdown.gettalong.org/documentation.html) superset of Markdown (`.md`) and should:
```